UKHomeOffice/sgmr-service
# SPCR-480 Fix port invalid bug

### AC / Description of changes
To recreate:
- create a voyage
- fill in and submit the arrival page
- go back from arrivals to arrival page

Expected behaviour:
- page is pre-filled
- you can submit again without making changes

Observed behaviour:
- page is pre-filled
- port will error as though it hadn’t been completed ‘you must enter a port’

### To Test
- create a voyage
- fill in and submit the arrival page
- go back to arrival page
- page should be pre-filled
- > you should be able to submit again without making changes

### Notes

---

**Remember to merge to feature branches not `master`.**
